stylesheets/style.css

```diff
@@ -0,0 +1,618 @@
+}
+
+.horrizontal-line {
+  display: flex;
+}
+
+.search-bar,
+.name {
+  margin-left: 25px;
+}
+
+.name {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+aside img {
+  width: 75px;
+  height: 75px;
+  padding: 10px 10px 10px 0;
+}
+
+.logo img {
+  height: 42px;
+  width: 220px;
+  margin-top: -20px;
+  margin-left: 300px;
+}
+
+nav button {
+  border-radius: 3px;
+  cursor: pointer;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  transition: all 0.6s ease 0s;
+  white-space: nowrap;
+  vertical-align: middle;
+  background-color: rgb(86, 123, 149);
+  border: 1px solid rgb(50, 104, 145);
+  color: rgb(255, 255, 255);
+  font-size: 11px;
+  line-height: 11px;
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  padding: 11px 12px 8px;
+  text-transform: uppercase;
+}
+
+.subscribe button {
+  margin-left: 280px;
+}
+
+.subscribe button,
+.log-in button {
+  margin-top: -6px;
+}
+
+.log-in button {
+  margin-left: 6px;
+}
+
+.subscribe button,
+.log-in button,
+.section-footer button {
+  height: 30px;
+  background-color: #567b95;
+  border-radius: 4px;
+  border: 1px solid;
+  color: white;
+}
+
+.center {
+  text-align: center;
+}
+
+.headline-1 {
+  padding-top: 90px;
+  font-family: 'Crimson Text', serif;
+  font-style: italic;
+  font-weight: 700;
+  font-size: xx-large;
+  width: 1250px;
+}
+
+.image-set img {
+  padding-top: 20px;
+  height: 450px;
+  width: 75%;
+}
+
+.space-around {
+  justify-content: space-around;
+}
+
+.title-credits {
+  width: 70%;
+  color: #798795;
+  margin-right: auto;
+  text-align: justify;
+  font-size: 1.2rem;
+}
+
+.date-time {
+  font-size: 0.7rem;
+}
+
+.same-line {
+  display: flex;
+}
+
+.social-media-w {
+  width: 590px;
+}
+
+.social-media {
+  margin-left: 300px;
+  font-size: large;
+  display: flex;
+}
+
+.social-media i {
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 15px;
+  background-color: lightgray;
+  border-radius: 80%;
+  height: 30px;
+  width: 20px;
+  cursor: pointer;
+  margin-left: 5px;
+  margin-right: 5px;
+}
+
+.social-media i:hover {
+  background-color: white;
+  border: 1px solid lightgray;
+}
+
+.article-p {
+  min-width: 600px;
+  max-width: 600px;
+  padding-top: 50px;
+  font-family: 'Crimson Text', serif;
+  font-size: x-large;
+}
+
+.info {
+  padding-bottom: 20px;
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+  margin-bottom: 10px;
+}
+
+.info img {
+  min-width: 600px;
+  max-width: 600px;
+}
+
+.thanks {
+  max-width: 300px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+
+.thanks p {
+  font-size: medium;
+  font-weight: 700;
+  padding-left: 30px;
+}
+
+.a-nodecoration {
+  text-decoration: none;
+}
+
+.black {
+  color: black;
+}
+
+.thanks-line {
+  width: 100px;
+  color: black;
+  margin: 20px auto;
+}
+
+main {
+  width: 940px;
+}
+
+@media (min-width: 600px) and (max-width: 800px) {
+  .name {
+    display: none;
+  }
+
+  .aside {
+    display: none;
+  }
```

Comment:
DRY stands for "Don't Repeat Yourself". Keeping the project code dry helps readability and makes it easier to find bugs. Example of how to keep the code DRY.
Improved version of the code above:
```
  .name, 
.aside {
    display: none;
  }
```